core/src/modules/cloudCli.ts

```diff
@@ -119,11 +119,11 @@ export namespace CloudCLI {
     const tempPath = await mkdtemp(path.join(os.tmpdir(), "grouparoo-cli-"));
     const archivePath = path.join(tempPath, "grouparoo.tar.gz");
 
-    GrouparooCLI.logger.log("Building config archive...\n");
+    GrouparooCLI.logger.log("\n🛠 Building config archive...\n");
 
     await CloudCLI.pack(archivePath);
 
-    GrouparooCLI.logger.log("\nUploading config...\n");
+    GrouparooCLI.logger.log("\n☁️ Uploading config...\n");
```

Comment:
note: i saw this on my phone yesterday an it looked like a cloud- now on my computer it looks like this 
![image](https://user-images.githubusercontent.com/4368928/143039400-285a90de-b91d-479e-ab87-55aa84e5cbcd.png)

🤷🏼

Reply:
Same... I'm happy with the icon either way though